mender-deb-package

```diff
@@ -240,7 +230,11 @@ EOF
     rm -f debian/patches/series
   fi
 
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6ED0E7B82643E131 2> /dev/null
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0E98404D386FA1D9 2> /dev/null
```

Comment:
I guess you have gone back and forth with this already, but I cannot still understand why this cannot be done at built time.

Reply:
Also, `apt-key` is deprecated, we should use `trusted.gpg.d`. Google a bit on your side and see a similar fix we did some time ago here: https://github.com/mendersoftware/mender-dist-packages/commit/7d4341b218e9d62c5d9f4e4c0ed8a5d7605104f1